deploy-manage/monitor/autoops/autoops-stack-monitoring-comparison.md

```diff
@@ -0,0 +1,154 @@
+---
+applies_to:
+  deployment:
+    ess: all
+    ece: all
+    eck: all
+    self: all
+navigation_title: AutoOps and Stack Monitoring comparison
+---
+
+# AutoOps and Stack Monitoring comparison
+
+This page provides a detailed comparison of AutoOps and Stack Monitoring to help you decide which solution is better suited to your needs.
+
+## Review the main differences
+Review how both tools differ in their provisioning, setup, access, and investigative abilities.
+
+### Resource provisioning and billing
+
+#### AutoOps [ao-resource]
+AutoOps stores and backs up your monitoring data internally on {{ecloud}} infrastructure so you don’t need to think about provisioning, sizing, and availability. The data is retained for four days by default. Using AutoOps is free for {{ecloud}} customers and is offered to [all subscription tiers](https://www.elastic.co/subscriptions/cloud).
+
+#### Stack Monitoring [sm-resource]
+With Stack Monitoring, you are responsible for storing your monitoring data. This means provisioning the necessary resources based on your performance and retention needs as well as paying for the allocated resources. The default retention period is six days.
+
+### Setup
+
+#### AutoOps [ao-setup]
+On {{ech}} (ECH), AutoOps is set up and enabled automatically in all [supported regions](ec-autoops-regions.md) and there is no action required from you.
+
+:::{note}
+AutoOps will be available in self-managed environments in the future.
+:::
+
+:::{image} /deploy-manage/images/cloud-autoops-setup.png
+:alt: Diagram showing AutoOps setup in Elastic Cloud
+:::
+
+#### Stack Monitoring [sm-setup]
+Stack Monitoring is a Kibana application that can be enabled on self-managed clusters on your premises, ECH deployments, {{eck}} (ECK), and {{ece}} (ECE). Stack Monitoring is not available on {{serverless-full}} since Elastic takes care of monitoring and managing your Serverless projects.
```

Comment:
```suggestion
Stack Monitoring is a {{kib}} application that can be enabled on self-managed clusters on your premises, ECH deployments, {{eck}} (ECK), and {{ece}} (ECE). Stack Monitoring is not available on {{serverless-full}} since Elastic takes care of monitoring and managing your Serverless projects.
```